### ISSUE TITLE
Installing packages requires user input

### ISSUE THREAD

**Issue by yiendos** [joomlatools/joomlatools-vagrant#156]
Seems like the most recent developments to the vagrant box, has left the box unable to install packages correctly: 

```
error with automatic installing of packages most probably need a -y flag


    default: Preparing to unpack .../10-ruby-net-telnet_0.1.1-2_all.deb ...
    default: Unpacking ruby-net-telnet (0.1.1-2) ...
    default: Selecting previously unselected package ruby-power-assert.
    default: Preparing to unpack .../11-ruby-power-assert_0.3.0-1_all.deb ...
    default: Unpacking ruby-power-assert (0.3.0-1) ...
    default: Selecting previously unselected package ruby-test-unit.
    default: Preparing to unpack .../12-ruby-test-unit_3.2.5-1_all.deb ...
    default: Unpacking ruby-test-unit (3.2.5-1) ...
    default: Selecting previously unselected package libruby2.5:amd64.
    default: Preparing to unpack .../13-libruby2.5_2.5.1-1ubuntu1.5_amd64.deb ...
    default: Unpacking libruby2.5:amd64 (2.5.1-1ubuntu1.5) ...
    default: Package configuration┌──────────────────────┤ Configuring libssl1.1:amd64 ├──────────────────────┐│││ There are services installed on your system which need to be restarted    ││ when certain libraries, such as libpam, libc, and libssl, are upgraded.   ││ Since these restarts may cause interruptions of service for the system,   ││ you will normally be prompted on each upgrade for the list of services    ││ you wish to restart.  You can choose this option to avoid being││ prompted; instead, all necessary restarts will be done for you││ automatically so you can avoid being asked questions on each library││ upgrade.││││ Restart services during package upgrades without asking?││││<Yes><No>│││└───────────────────────────────────────────────────────────────────────────┘
^[[A^[[Dult: Processing triggers for mime-support (3.60ubuntu1) ...
    default: Setting up unzip (6.0-21ubuntu1) ...
    default: Setting up zip (3.0-11build1) ...
    default: Setting up fonts-lato (2.0-2) ...
    default: Setting up ruby-did-you-mean (1.2.0-2) ...
    default: Processing triggers for libc-bin (2.27-3ubuntu1) ...
    default: Setting up libssl1.1:amd64 (1.1.1-1ubuntu2.1~18.04.4) ...
    default: Checking for services that may need to be restarted...
    default: done.
    default: Checking for services that may need to be restarted...
    default: done.
    default: Checking init scripts...
    default:
```

Which then requires manual intervention on the box: 

```
vagrant into the box 

`ps aux | grep -i apt` 

then kill and processes 

then proceed to configure any outstanding packages via 

 `sudo dpkg --configure -a`

 You might need to be root 

 `sudo passwd root`

 su 
```

**Comment by stevenrombauts** [comment]
@yiendos Is this during the initial `vagrant up` command? During which stage exactly?

**Comment by stevenrombauts** [comment]
@yiendos This is fixed in 73e6b7d
